package/amazon-alexa/apl-client-library/0002-install-apl-client-js.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2d1d387..f716f1f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -15,4 +15,8 @@ add_subdirectory("APLClient")
+ 
+ if(SANDBOX)
+     add_subdirectory("APLClientSandbox")
++
++if (CMAKE_INSTALL_PREFIX)
```

Comment:
Is that if needed? I would just do an install, as it's needed in all cross compiled environment

Reply:
It is installing APL client libraries or includes also if CMAKE_INSTALL_PREFIX is set https://github.com/alexa/apl-client-library/blob/main/APLClient/src/CMakeLists.txt#L72. Other wise it will try to install on /share